src/lib/components/Modal/README.mdx

```diff
@@ -994,9 +994,9 @@ opened.
 | `--rui-Modal__padding-y`                             | Block padding of individual modal components                 |
 | `--rui-Modal__background`                            | Modal background (including `url()` or gradient)             |
 | `--rui-Modal__box-shadow`                            | Modal box shadow                                             |
-| `--rui-Modal__separator__width`                      | Width of separator between modal head, body, and footer      |
-| `--rui-Modal__separator__color`                      | Color of separator between modal head, body, and footer      |
-| `--rui-Modal__head__gap`                             | Modal head gap between children                              |
+| `--rui-Modal__separator__width`                      | Width of separator between modal header, body, and footer    |
+| `--rui-Modal__separator__color`                      | Color of separator between modal header, body, and footer    |
+| `--rui-Modal__header__gap`                           | Modal header gap between children                            |
```

Comment:
Could you please move this line above `--rui-Modal__footer__background`? I think I messed up the order doing the redesign… 😐 Thanks!